src/Standards/ISAAC/Sniffs/Classes/PropertyPerClassLimitSniff.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace IsaacCodingStandard\Standards\ISAAC\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use SlevomatCodingStandard\Helpers\PropertyHelper;
+use SlevomatCodingStandard\Helpers\ScopeHelper;
+use SlevomatCodingStandard\Helpers\SniffSettingsHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+use function count;
+use function sprintf;
+
+use const T_ANON_CLASS;
+use const T_CLASS;
+use const T_TRAIT;
+use const T_VARIABLE;
+
+class PropertyPerClassLimitSniff
+{
+    public const CODE_PROPERTY_PER_CLASS_LIMIT = 'PropertyPerClassLimit';
+
+    public const ERROR_MESSAGE = '%s has too many properties: %d. Can be up to %d properties.';
+
+    /** @var int */
+    public $maxPropertyCount = 10;
+
+    /**
+     * @return array<int, (int|string)>
+     */
+    public function register(): array
+    {
+        return [T_CLASS, T_TRAIT, T_ANON_CLASS];
+    }
+
+    /**
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
```

Comment:
I think this can be removed. I removed this line and could ran `vendor/bin/phpcs` without errors.

Reply:
Removed.

However, when running phpcs using PHP 7.4 or 8.0 a `MissingNativeTypeHint` violation was reported for another line, so I added that.